src/components/composer/autocomplete_dropdown/autocomplete_dropdown.xml

```diff
@@ -4,7 +4,7 @@
       t-ref="autoCompleteList"
       t-att-class="{
           'o-autocomplete-dropdown': props.proposals.length}">
-      <t t-foreach="props.proposals" t-as="proposal" t-key="proposal.text">
+      <t t-foreach="props.proposals" t-as="proposal" t-key="proposal_index">
```

Comment:
I just asked Géry because I wasn't entirely sure because in practice it looks like this is working but...

*les clés _index, c'est uniquement ok si tu es sur a l'avance que ta liste est statique (ou éventuellement, si on ajoute des éléments, mais sans en enlever)
autrement, tu vas réconcilier des élém différents, et en général ca va marcher, en surface, sauf que ca risque de faire des trucs bizarres. par exemple, si l'utilisateur écrit dans un input, l'input va etre associé a un autre élément*

Reply:
maybe a composite key would do the trick without the shortcomings of just `_index`
```suggestion
      <t t-foreach="props.proposals" t-as="proposal" t-key="proposal.text + proposal_index">
```

Reply:
yup forgot that the proposals change over time ><